source/includes/_event_subscriptions.md

```diff
@@ -83,6 +83,11 @@ Events currently supported are:
 
 ### RAMS
 - `version_pdf_created` - this is fired after we have generated and stored a project version PDF, which is now ready to be downloaded. Please note that the event is scoped to look for subscriptions which were created by the same division that the project belongs to.
+- `project_archived` - this is fired after a project is archived (via the HandsHQ app or via the API).
+- `project_unarchived` - this is fired after a project is unarchived (via the HandsHQ app or via the API).
+- `personnel_assignment_created` - this is fired after a personnel has been added to a project (via the HandsHQ app or via the API).
+- `personnel_assignment_updated` - this is fired after a personnel role has been updated on a project (via the HandsHQ app or via the API).
```

Comment:
I don't think we should make this specific to role, it should just be about the details of the assignment of a personnel to a project - though I think it's fine as an example of an update. I know that's the only thing that can change atm but that can reasonably change in the future